client/state/selectors/get-current-locale-slug.js

```diff
@@ -1,24 +1,13 @@
 /**
- * Eternal dependencies
- *
- * @format
+ * External dependencies
  */
-
 import { get } from 'lodash';
 
-/**
- * Internal dependencies
- */
-import createSelector from 'lib/create-selector';
-
 /**
  * Gets the current ui locale slug
  * @param {Object} state - global redux state
  * @return {String} current state value
  */
-const getCurrentLocaleSlug = createSelector(
-	state => get( state, 'ui.language.localeSlug', null ),
-	state => state.ui.language
-);
-
-export default getCurrentLocaleSlug;
+export default function getCurrentLocaleSlug( state ) {
+	return get( state, 'ui.language.localeSlug', null );
```

Comment:
Nit: would it be worth turning this into an array of strings, to avoid string parsing?

Reply:
Or, of course, optional chaining + nullish operator. Although I don't think we can use that yet, right?

```js
return state?.ui?.language?.localeSlug ?? null;
```

Reply:
Lodash caches the parsed array internally, so it's not a big performance win. Let's merge the PR as-is and hopefully we'll be able to convert it to optional chaining soon. I think that if we upgraded to latest TypeScript and added the right plugin to the Babel config, we could use it really soon!